xmodule/video_block/video_handlers.py

```diff
@@ -535,7 +536,9 @@ def studio_transcript(self, request, dispatch):
                             'edx_video_id': edx_video_id,
                             'language_code': new_language_code
                         }
-                        self.transcripts[new_language_code] = f'{edx_video_id}-{new_language_code}.srt'
+                        if language_code != new_language_code:
+                            self.transcripts.pop(language_code, None)
```

Comment:
you have the same check in the `add_or_update_transcripts` function, why do you duplicate it here?

Reply:
@dyudyunov Right, thanks for noticing this redundant code. Initially, I wanted to do it through a separate function. But then I realized that it was redundant. And after moving the logic to studio_transcript I forgot to delete add_or_update_transcripts.

Reply:
I removed the add_or_update_transcripts function.